Exercise_B/assets/style/stylesheet.css

```diff
@@ -1 +1,18 @@
 /* Your CSS code here */
+
+html {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.card img {
+    filter: brightness(30%);
+}
+
+.font-weight-bold {
+    font-weight: normal;
+}
+
+#green {
+    background-color: #cee7b4;

```

Comment:
Might be useful to have the font a different colour than the background, if you do then you wont be able to see the text.